docs/root/start/quick-start/configuration-dynamic-filesystem.rst

```diff
@@ -8,6 +8,11 @@ protocol.
 
 When the files are changed on the filesystem, Envoy will automatically update its configuration.
 
+.. note::
+
+   Now Enovy only get the changes when write to temporary file and rename into place, ignore overwritten directly.
+   The purpose is  that atomic renames are the only way to achieve consistency.
```

Comment:
The grammar is not quite correct here and (esp for a beginner doc) this is not so easy to follow

Suggested change:

> Envoy only updates when the configuration file is replaced, and not when the file is edited in place. It is implemented this way to ensure configuration consistency.

Reply:
Please also ref link to this section of the docs, which is a general explanation of how the atomic swap works: https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/runtime#updating-runtime-values-via-symbolic-link-swap

Reply:
@phlax and @mattklein123, thanks for the review. I have updated.